static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -83,3 +86,67 @@ export function getDiseaseGeneAssociation(
   }
   return rawData;
 }
+/**
+ * Fetches the disease-symptom association data
+ * @param data
+ * @returns
+ */
+export function getDiseaseSymptomAssociation(
+  data: GraphNodes
+): DiseaseSymptomAssociationData[] {
+  // checks if the data is empty
+  if (_.isEmpty(data)) {
+    return [];
+  }
+  const rawData: DiseaseSymptomAssociationData[] = [];
+  // checks for null values
+  if (_.isEmpty(data.nodes) || _.isEmpty(data.nodes[0].neighbors)) {
+    return [];
+  }
+  for (const neighbour of data.nodes[0].neighbors) {
+    if (neighbour.property !== "diseaseOntologyID") {
+      continue;
+    }
+    for (const node of neighbour.nodes) {
+      let symptom = null;
+      let oddsRatioValue = null;
+      // check for null or non-existent property values
+      if (_.isEmpty(node.neighbors)) {
+        continue;
+      }
+      for (const n of node.neighbors) {
+        if (n.property === "associationOddsRatio") {
+          // check for empty list and null gene values
+          if (_.isEmpty(n.nodes) || _.isEmpty(n.nodes[0].value)) {
+            continue;
+          }
+          oddsRatioValue = n.nodes[0].value;
+        } else if (n.property === "medicalSubjectHeadingID") {
+          for (const n1 of n.nodes) {
+            if (n1.neighbors === undefined || _.isEmpty(n1.neighbors)) {
```

Comment:
_.isEmpty() checks for undefined so you don't need that additional `=== undefined` check